src/hotspot/share/opto/castnode.hpp

```diff
@@ -78,6 +78,11 @@ class CastIINode: public ConstraintCastNode {
     : ConstraintCastNode(n, t, carry_dependency), _range_check_dependency(range_check_dependency) {
     init_class_id(Class_CastII);
   }
+  CastIINode(Node* ctrl, Node* n, const Type* t, bool carry_dependency = false, bool range_check_dependency = false)
```

Comment:
Not used.

Reply:
Done, this will gracefully limit the value range of length and also constrain scheduling of VectorMaskGen.